src/graphql.tsx

```diff
@@ -22,6 +22,7 @@ import ApolloClient, {
   ApolloStore,
   ApolloQueryResult,
   ApolloError,
+  // FetchPolicy,
```

Comment:
Unfortunately it's not exported by Apollo Client yet. "Someone" forgot to export it. We'll add it in the next release.